packages/gatsby/src/commands/develop.ts

```diff
@@ -75,7 +75,13 @@ class ControllableScript {
       stdio: [`inherit`, `inherit`, `inherit`, `ipc`],
     })
   }
-  async stop(signal: string | null = null, code?: number): Promise<void> {
+  async stop(
+    signal: NodeJS.Signals | null = null,
+    code?: number
+  ): Promise<void> {
+    if (!this.process) {
+      throw new Error(`Trying to stop process before starting it`)
```

Comment:
Add an article `the`

```suggestion
      throw new Error(`Trying to stop the process before starting it`)
```